packages/@apphosting/adapter-angular/src/bin/build.ts

```diff
@@ -1,9 +1,21 @@
 #! /usr/bin/env node
-import { checkBuildConditions, build, generateOutputDirectory } from "../utils.js";
+import { checkBuildConditions, build, generateOutputDirectory, DEFAULT_COMMAND } from "../utils.js";
 
-const cwd = process.cwd();
+const root = process.cwd();
 
-await checkBuildConditions(cwd);
+// Determine root of project to build
+let projectRoot = root;
+if (process.env.MONOREPO_PROJECT && process.env.FIREBASE_APP_DIRECTORY) {
+  projectRoot = projectRoot.concat("/", process.env.FIREBASE_APP_DIRECTORY);
+} else {
+  await checkBuildConditions(projectRoot);
+}
 
-const outputBundleOptions = await build().catch(() => process.exit(1));
```

Comment:
Why was the .catch(() => process.exit(1)) removed here?

Reply:
Oh good catch - I removed this while debugging because the program would silently exit without providing the specific error message. If we are intentionally hiding the error, I can add this back in.

Reply:
Ah gotcha, @sjjj986 do you remember why we needed this catch? Do you think its safe to remove?

Reply:
looks good to me